testhelpers/testhelpers.go

```diff
@@ -580,3 +580,18 @@ func writeTAR(t *testing.T, srcDir, tarDir string, mode int64, w io.Writer) {
 	err := archive.WriteDirToTar(tw, srcDir, tarDir, 0, 0, mode, true, nil)
 	AssertNil(t, err)
 }
+
+func CreateAndAssignTempPackHome(t *testing.T) (string, error) {
```

Comment:
On second thought, this helper is probably unnecessary...

Reply:
The downside to this helper is that is tries to hide the need to know that `PACK_HOME`  should be set but you need to remember to unset it (or even better set it to the previously set value). I can see the value of a helper struct more than a single method. One that lets you register and unregister. That said, it's value is relatively small.